build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql

```diff
@@ -198,3 +198,17 @@ ALTER TABLE flows
 ALTER TABLE flows_local
     DROP COLUMN egressName,
     DROP COLUMN egressIP;
+ALTER TABLE tadetector
+    DROP COLUMN podNamespace;
```

Comment:
It should be a comma instead of semicolon between DROP COLUMNs, please check my previous comment: https://github.com/antrea-io/theia/pull/184#discussion_r1244471976